argparse/argparse.py

```diff
@@ -140,11 +151,9 @@ def parse_args(self, args=None):
 
     def _parse_args(self, args):
         # add optional args with defaults
-        arg_dest = []
```

Comment:
ack. I agree with your preference. When I started making changes to this module I thought having one _Arg() instance per option would result in less changes and make the PR easier to accept. I will look into changing _Arg() the way you suggest.